extras/dht/dht.c

```diff
@@ -0,0 +1,128 @@
+

```

Comment:
Please add a copyright/license notice here and on the header file.

Something like these 3 lines is fine:
https://github.com/SuperHouse/esp-open-rtos/blob/master/extras/dhcpserver/dhcpserver.c#L12

(With the Copyright line changed as suitable.)